src/bp/nlu-core/engine.ts

```diff
@@ -171,8 +171,7 @@ export default class Engine implements NLU.Engine {
     }
 
     if (!trainAllCtx) {
-      model.data.output = _.merge({}, previousModel.data.output, model.data.output)
-      model.data.output.slots_model = new Buffer(model.data.output.slots_model) // lodash merge messes up buffers
```

Comment:
This isn't breaking anymore because we don't use lodash merge anymore, correct ?

Reply:
yes, I already double checked it!